### DISCUSSION TITLE
Get current date without dependent on systems date as well as it doesnt depend on internt

### DISCUSSION THREAD

**Discussion by abhijitpatil181171** [js-fns/date-fns#3683] [Q&A]
Currently i am working on node js backend . my client side and server on same machine locally what i want i want to get current date without depending on systems date and also it doesnt depend on internet . is it possible in js

**Comment by joaopedrodcf** [comment]
Maybe this can help you basically in epic-stack they discussed an implemented exactly that: https://github.com/epicweb-dev/epic-stack/blob/main/docs/timezone.md

Client preferred cookies: https://www.epicweb.dev/tips/use-client-hints-to-eliminate-content-layout-shift

Basically the concept is the client sends you cookies to the server with their preferred options: https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/005-client-pref-cookies.md
